tasks/breeze/generators/templates/breeze_config.ecr

```diff
@@ -6,4 +6,8 @@ Breeze.configure do |settings|
 
   # Enable Breeze only for this environment
   settings.enabled = Lucky::Env.development?
+
+  # Set this to your Carbon::EmailPreviews class
+  # This should be defined in src/emails/previews.cr
+  # settings.email_previews = Emails::Previews
```

Comment:
This probably shouldn't be in here? Because this is the default config.. There's no telling if you're going to use this setup or not..

Reply:
Since this extension will be baked-in, even though it's optional, I just went with adding more comments to it.